### PR TITLE
Refactor countries page

### PR DESCRIPTION
**Pull request by MattiSG**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Thank you for wanting to contribute to OpenFisca! :smiley:  
+Thank you for wanting to contribute to OpenFisca! :smiley:
 
-Feel free to contact us or send us GIT pull requests.  
+Feel free to contact us or send us pull requests over GitHub.
```
Comment (maukoquiroga) [inline CONTRIBUTING.md]: Maybe `on` or `via` Github?

**Comment by maukoquiroga** [comment]
Hello @MattiSG, thanks for your contribution. Could you rebase please ?

**Comment by maukoquiroga** [comment]
Poke @MattiSG

**Comment by maukoquiroga** [comment]
I'll rebase this myself if you don't mind @MattiSG

**Comment by MattiSG** [comment]
Please do!

**Pull request merged**